tests/aws/services/events/test_events.py

```diff
@@ -568,6 +568,137 @@ def test_create_connection_validations(self, aws_client):
         assert "must have length less than or equal to 64" in message
         assert "must satisfy enum value set: [BASIC, OAUTH_CLIENT_CREDENTIALS, API_KEY]" in message
 
+    @markers.aws.validated
+    def test_put_events_response_entries_order(
+        self, events_put_rule, sqs_create_queue, sqs_get_queue_arn, aws_client, snapshot, clean_up
+    ):
+        """Test that put_events response contains each EventId only once, even with multiple targets."""
+        account_id = aws_client.sts.get_caller_identity()["Account"]
+        region = aws_client.events.meta.region_name
+
+        # Create two SQS queues as targets
+        queue_url_1 = sqs_create_queue()
+        queue_arn_1 = sqs_get_queue_arn(queue_url_1)
+        queue_url_2 = sqs_create_queue()
+        queue_arn_2 = sqs_get_queue_arn(queue_url_2)
+
+        rule_name = f"test-rule-{short_uid()}"
+
+        snapshot.add_transformers_list(
+            [
+                snapshot.transform.key_value("EventId", reference_replacement=False),
+                snapshot.transform.regex(queue_arn_1, "<queue-1-arn>"),
+                snapshot.transform.regex(queue_arn_2, "<queue-2-arn>"),
+                snapshot.transform.regex(rule_name, "<rule-name>"),
+                *snapshot.transform.sqs_api(),
+                *snapshot.transform.sns_api(),
+            ]
+        )
+
+        rule_arn = f"arn:aws:events:{region}:{account_id}:rule/{rule_name}"
+        sqs_policy = {
+            "Version": "2012-10-17",
+            "Statement": [
+                {
+                    "Sid": "EventBridgeToSQS",
+                    "Effect": "Allow",
+                    "Principal": {"Service": "events.amazonaws.com"},
+                    "Action": "sqs:SendMessage",
+                    "Resource": [queue_arn_1, queue_arn_2],
+                    "Condition": {"ArnLike": {"aws:SourceArn": rule_arn}},
+                }
+            ],
+        }
+
+        aws_client.sqs.set_queue_attributes(
+            QueueUrl=queue_url_1, Attributes={"Policy": json.dumps(sqs_policy)}
+        )
+        aws_client.sqs.set_queue_attributes(
+            QueueUrl=queue_url_2, Attributes={"Policy": json.dumps(sqs_policy)}
+        )
+
+        events_put_rule(
+            Name=rule_name,
+            EventPattern=json.dumps(
+                {"source": ["test.source"], "detail-type": ["test.detail.type"]}
```

Comment:
it would make sense to use the constant pattern here too: `TEST_EVENT_PATTERN_NO_DETAIL` from `test_events.py` and similarly use `TEST_EVENT_PATTERN` as test event in line 647